spk/umurmur/src/gencert.sh

```diff
@@ -2,19 +2,18 @@
 
 # Variables
 PACKAGE="umurmur"
-INSTALL_DIR="/usr/local/${PACKAGE}"
-OPENSSL="${INSTALL_DIR}/bin/openssl"
+OPENSSL="${SYNOPKG_PKGDEST}/bin/openssl"
 
 # Certificate generation
-${OPENSSL} req -x509 -newkey rsa:1024 -keyout ${INSTALL_DIR}/var/umurmur.key -nodes -sha1 -days 365 -out ${INSTALL_DIR}/var/umurmur.crt -batch -config ${INSTALL_DIR}/openssl.cnf > /dev/null 2>&1
+${OPENSSL} req -x509 -newkey rsa:1024 -keyout ${SYNOPKG_PKGDEST}/var/umurmur.key -nodes -sha1 -days 365 -out ${SYNOPKG_PKGDEST}/var/umurmur.crt -batch -config ${SYNOPKG_PKGDEST}/openssl.cnf > /dev/null 2>&1
 
 # Exit with the right code and an explicit message
 if [ $? -ne 0 ]; then
-    echo "Certificate generation for uMurmur failed"
-    touch ${INSTALL_DIR}/var/umurmur.key
-    touch ${INSTALL_DIR}/var/umurmur.crt
+    echo "Certificate generation for uMurmur failed" >> ${INST_LOG}
+    touch ${SYNOPKG_PKGDEST}/var/umurmur.key
+    touch ${SYNOPKG_PKGDEST}/var/umurmur.crt
     exit 1
 fi
 
-echo "uMurmur's certificate successfully generated"
+echo "uMurmur's certificate successfully generated" >> ${INST_LOG}
```

Comment:
I support script is expected to be also started out of installer `postinst`, so I would remove redirection to `INST_LOG` that may not be defined.